Add Google Tag Manager tracking, refs #13284

replaceafill opened the pull request:
- Add config setting in 'config/app.yml':
    
Enables tracking through a container ID. See
https://developers.google.com/tag-manager/quickstart
    
- Add tag manager component:
    
Called from all the layout templates (except install) to render the
script and noscript snippets at the suggested points in the HTML.

mcantelon commented on apps/qubit/modules/default/actions/tagManagerComponent.class.php:
```diff
@@ -0,0 +1,30 @@
+  public function execute($request)
+  {
+    $config = sfConfig::get('app_google_tag_manager_container_id', '');
+    if (empty($config) || !isset($this->code)) {
```
Nitpick: Can code be an empty string? If not, using `!empty($this->code)` would guard again that.

mcantelon reviewed (approved):
One nitpick, but looks good otherwise!

replaceafill committed:
Add Google Tag Manager tracking, refs #13284

- Add config setting in 'config/app.yml':

Enables tracking through a container ID. See
https://developers.google.com/tag-manager/quickstart

- Add tag manager component:

Called from all the layout templates (except install) to render the
script and noscript snippets at the suggested points in the HTML.

replaceafill commented:
Thank you @mcantelon!
